app/app.go

```diff
@@ -857,7 +857,10 @@ func NewDesmosApp(
 		slashing.NewAppModule(appCodec, app.SlashingKeeper, app.AccountKeeper, app.BankKeeper, app.StakingKeeper),
 		params.NewAppModule(app.ParamsKeeper),
 		evidence.NewAppModule(app.EvidenceKeeper),
-		authzmodule.NewAppModule(appCodec, app.AuthzKeeper, app.AccountKeeper, app.BankKeeper, app.interfaceRegistry),
+
+		// TODO: Re-enable this once the sim tests are fixed
+		// authzmodule.NewAppModule(appCodec, app.AuthzKeeper, app.AccountKeeper, app.BankKeeper, app.interfaceRegistry),
```

Comment:
Unfortunately, with version `0.45.9` of the SDK the simulation tests related to the `authz` module are failing due to incorrect generation